### DISCUSSION TITLE
Texture not supported by Model Viewer

### DISCUSSION THREAD

**Discussion by rocket9988** [google/model-viewer#3352] [Q&A]
I have been trying few glb model and notice some model texture rendered nicely but some unable to render texture.

I have try those fail to render texture glb model on gltf viewer and it works fine.

May I know is there any texture mapping not being supported by model viewer?

**Comment by elalish** [comment]
I'm going to need a lot more detail to be able to answer your question. Please fill out the issue template! Repro links, screenshots, models, devices are all super important for us to understand what you're talking about.
